src/System.Management.Automation/engine/remoting/fanin/WSManTransportManager.cs

```diff
@@ -2597,7 +2597,16 @@ internal WSManAPIDataCommon()
 #endif
 
                 _handle = IntPtr.Zero;
-                ErrorCode = WSManNativeApi.WSManInitialize(WSManNativeApi.WSMAN_FLAG_REQUESTED_API_VERSION_1_1, ref _handle);
+
+                try
+                {
+                    ErrorCode = WSManNativeApi.WSManInitialize(WSManNativeApi.WSMAN_FLAG_REQUESTED_API_VERSION_1_1, ref _handle);
+                }
+                catch (DllNotFoundException)
+                {
+                    throw new PSRemotingTransportException(
+                        StringUtil.Format(RemotingErrorIdStrings.WSManClientDllNotAvailable));
```

Comment:
`StringUtil.Format(..)` is unnecessary as no arguments is required for the error message.